docs/lib/DEBUGGING.md

```diff
@@ -152,3 +184,54 @@ If you are using ST-LINK, please refer to the [STMicroelectronics website](http:
 
 If instead you want to connect your debugger to the JTAG port, you must wire the needed pins to your connector. This [guide](https://www.segger.com/admin/uploads/evalBoardDocs/AN00015_ConnectingJLinkToSTM32F429Discovery.pdf) explains how to do that in detail. The guide is specific to the J-Link connectors, but you can apply it to other connectors.
 
+## Debugging with Eclipse
+
+* Make sure [GNU ARM Eclipse plugins](http://gnuarmeclipse.sourceforge.net/updates) is installed, if not then install:
```

Comment:
Please change to "Make sure [GNU Arm Eclipse plugins](http://gnuarmeclipse.sourceforge.net/updates) are installed. You can install them with:"

Reply:
OK